sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/OtherTests.java

```diff
@@ -324,6 +340,54 @@ public void testWith() throws IOException, InterruptedException {
         Utilities.compileAndTestRust(BaseSQLTests.rustDirectory, false);
     }
 
+    @Test
+    public void testUDFWarning() throws IOException {
+        File file = createInputScript("CREATE FUNCTION myfunction(d DATE, i INTEGER) RETURNS VARCHAR",
```

Comment:
Reading these tests now, I now see why you have a constant for the UDF file name. Is it because you copy it into the generated crate as `udf.rs`? I guess it's also an assumption you can evolve later.